client/src/project/new/components/Visibility.tsx

```diff
@@ -256,49 +274,51 @@ const EditVisibility = ({
       ""
     );
 
-  if (!isLoadingProject && !isFetchingProject && !projectData)
-    return (
-      <WarnAlert dismissible={false}>
-        Knowledge Graph integration must be enabled to change visibility from
-        the RenkuLab web interface.
-      </WarnAlert>
-    );
+  const isLoadingSomething =
```

Comment:
Nitpick: feels more serious to simply call it `isLoading`

```suggestion
  const isLoading =
```

Reply:
Actually `isLoading` is already used for something else in that component :sweat_smile: 
I will rename that to `isLoadingMutation` and change this to `isLoadingAnything`. `isLoading` would be too confusing since there are 4 isLoading variables

Reply:
I would avoid "Something" and "Anything" because they do not help explain what is happening in the code. Something like `isLoadingQueries` or `isLoadingData` gives a hint about what we are loading.